config/application.rb

```diff
@@ -69,8 +69,11 @@ class Application < Rails::Application
       config.access_policies = '/etc/rmt/access_policies.yml'
 
       config.cache_config_file = '/var/lib/rmt/rmt-cache-trim.sh'
+      registry_cache_expiration_time = Settings[:registry].try(:token_expiration) || 480 # 8 hours in minutes
```

Comment:
Can we move these engine specific configs to the engine directory?

Reply:
Good idea